.concourse/kubecf-pool-reconciler.yaml.gomplate

```diff
@@ -45,7 +45,6 @@ jobs:
     trigger: true
   - get: catapult
   - task: deploy
-    privileged: true
```

Comment:
SInce I'm at it I thought about removing it from here too. Will need to redeploy this pipeline too.